tests/e2e/helpers/chimoney.ts

```diff
@@ -0,0 +1,136 @@
+import type { Page } from '@playwright/test';
+
+export const URLS = {
+  get login() {
+    return `${process.env.CHIMONEY_WALLET_ORIGIN}/auth/signin`;
+  },
+  get keyPage() {
+    return `${process.env.CHIMONEY_WALLET_ORIGIN}/interledger`;
+  },
+};
+
+type GetWalletAddressKeysResponse = {
+  status: string;
+  data: {
+    node: {
+      id: string;
+      jwk: { kid: string };
+      revoked: boolean;
+      createdAt: string;
+    };
+  }[];
+};
+
+export async function revokeKey(page: Page, jwkKeyId: string) {
+  await page.goto(URLS.keyPage);
+
+  // The auth token changes on each page load?!
+  const authToken = await page.evaluate(() => {
+    // Same as from src/content/keyAutoAdd/chimoney.ts
+    const getAuthToken = (): string => {
+      const getFirebaseAuthKey = () => {
+        for (let i = 0; i < sessionStorage.length; i++) {
+          const key = sessionStorage.key(i);
+          if (key?.startsWith('firebase:authUser:')) {
+            return key;
+          }
+        }
+      };
```

Comment:
Since it is the same function in here as well, lets create a helper that we can reuse.

Reply:
It's same, but in different context 😭 
That one runs in browser, this one has to run in `page.evaluate()` context - this function will have to get serialized into `page.evaluate()` context for reuse. I'll try if straightforward - last I checked wasn't.

Reply:
It works!!! I'm very happy!